runtime/sema/deployed_contract.go

```diff
@@ -95,3 +107,18 @@ const DeployedContractTypeCodeFieldName = "code"
 const deployedContractTypeCodeFieldDocString = `
 The code of the contract
 `
+
+const DeployedContractTypePublicTypesFunctionName = "publicTypes"
+const DeployedContractTypePublicTypesFunctionDocString = `
+	Returns an array of Type objects representing all public type declarations in this contract.
+`
+
+var DeployedContractTypePublicTypesFunctionType = NewSimpleFunctionType(
+	FunctionPurityView,
+	[]Parameter{},
```

Comment:
No need to allocate a slice, can just be `nil`

```suggestion
	nil,
```